alphabase/psm_reader/maxquant_reader.py

```diff
@@ -101,6 +101,7 @@ def __init__(self,
         keep_decoy = False,
         fixed_C57 = True,
         mod_seq_columns = ['Modified sequence'],
+        csv_sep = '\t',
```

Comment:
In the later version, we determine csv_sep based on the given tsv file, so we should skip this param